resources/en/dialogs/nlg-strings.dialog.bxb

```diff
@@ -76,18 +76,103 @@ template-macro-def (SPACE_RESORT_RESULT) {
   params {
     param (result) {
       type (SpaceResort)
-      min (Required) max (One)
+      min (Required) max (Many)
     }
   }
   content {
     if (size(result) == 1) {
-      template ("Check out this #{concept(result)}.")
+      if ($can.device == 'bixby-mobile') {
+        template ("Check out this #{concept(result)}.")        
+      } else-if ($can.device == 'bixby-speaker') {
+        template ("#{macro('SPACE_RESORT_NAME_WITH_EMPHASIS', result)} #{macro('SPACE_RESORT_DETAILS', result)}")        
+      }
```

Comment:
no message for the other devices?

Reply:
Nope. Since we don't define any other device types as targets in the capsule.bxb file, these are the only cases. It's easier to read this way, and when we add new targets we should verify all the flows anyways.